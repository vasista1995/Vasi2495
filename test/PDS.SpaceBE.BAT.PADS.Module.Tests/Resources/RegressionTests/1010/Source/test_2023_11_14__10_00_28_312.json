{
  "IdSource": "SPACEACT2:BAT:BE:ZA332675B36:BE1I:Wire_Bond1:CopperBallDiameterCTDownbond:70005:240189655:0",
  "Data_flat_Metadata": {
    "Device": "U5886B",
    "ParameterClass": "NON_SPC",
    "CFComment": "FOL",
    "Segment": "WB",
    "FourDReport": "*",
    "ParameterName": "CopperBallDiameterCTDownbond",
    "ParameterUnit": "um",
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "70005",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-11-14T07:28:02Z",
    "CreatedTimestamp": "2023-11-14T14:28:02Z",
    "UpdatedTimestamp": "2023-11-14T14:28:02Z",
    "UpdatedTimestampUtc": "2023-11-14T07:28:02Z",
    "ExportedTimestampUtc": "2023-11-14T14:28:02Z",
    "ExportedTimestamp": "2023-11-14T14:28:02Z",
    "SampleTimestampUtc": "2023-11-14T07:28:02Z",
    "SampleTimestamp": "2023-11-14T14:28:02Z",
    "SourceDataLevel": "C",
    "ChannelName": "Ball_Diameter/CU/30um/CT_Downbond/LSS/Bump",
    "ChannelDescr": "Bump Diameter, Downbond",
    "RvStoreFlag": "Y",
    "CalcCreated": "05/03/2021 15:56:44",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 1315368,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "1010",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 105.0,
        "SpecLow": 75.0,
        "SpecTarget": 90.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYNNNNNNNNNNNNNNNNNNNNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "90,92",
        "Median": "90,8",
        "Sigma": ",576194411635519",
        "Min": "90,3",
        "Max": "91,7",
        "Q1": "90,5",
        "Q3": "91,3"
      },
      "ControlLimits": {
        "CtrlMeanLowEnabled": "N",
        "CtrlMeanTargetEnabled": "N",
        "CtrlMeanHighEnabled": "N",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "CtrlRangeHighEnabled": "N",
        "ExtMaMV": 91.835,
        "ExtEwmaMV": 92.2371658239001,
        "ExtMsMV": 2.77185858225127,
        "EwmaS": 1.64928154377986,
        "EwmaR": 3.98699031470866,
        "ExtEwmaMVLCL": 0.0,
        "ExtEwmaMVCenter": 0.0,
        "ExtEwmaMVUCL": 0.0,
        "ExtMaMVLCL": 0.0,
        "ExtMaMVCenter": 0.0,
        "ExtMaMVUCL": 0.0,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.0,
        "ExtMSMVCenter": 0.0,
        "ExtMSMVUCL": 0.0,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 0.0,
        "RawUal": 0.0,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 240189655,
        "Seqnr": 4,
        "Value": 91.7,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240189655,
        "Seqnr": 3,
        "Value": 91.3,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240189655,
        "Seqnr": 2,
        "Value": 90.3,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240189655,
        "Seqnr": 1,
        "Value": 90.5,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 240189655,
        "Seqnr": 5,
        "Value": 90.8,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:Wire_Bond1:CopperBallDiameterCTDownbond:70005",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:ZA332675B36:BE1I:Wire_Bond1:CopperBallDiameterCTDownbond:70005:240189655:0"
  }
}