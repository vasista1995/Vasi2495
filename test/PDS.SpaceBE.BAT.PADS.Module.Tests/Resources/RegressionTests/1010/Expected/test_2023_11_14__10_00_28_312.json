{
  "Data_flat_Metadata": {
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-11-14T07:28:02Z",
    "BeginTimestamp": "2023-11-14T14:28:02Z",
    "EndTimestampUtc": "2023-11-14T07:28:02Z",
    "EndTimestamp": "2023-11-14T14:28:02Z"
  },
  "Data_flat_StrangeMetadata": {
    "Device": "U5886B",
    "ParameterClass": "NON_SPC",
    "Segment": "WB"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 4,
          "SampleId": 240189655,
          "Value": 91.7,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T14:28:02Z",
          "SampleTimestampUtc": "2023-11-14T07:28:02Z",
          "CreatedTimestamp": "2023-11-14T14:28:02Z",
          "CreatedTimestampUtc": "2023-11-14T07:28:02Z",
          "UpdatedTimestamp": "2023-11-14T14:28:02Z",
          "UpdatedTimestampUtc": "2023-11-14T07:28:02Z",
          "NumViolations": 0,
          "SampleMean": 90.92,
          "SampleMin": 90.3,
          "SampleMax": 91.7,
          "SampleMedian": 90.8,
          "SampleStdev": 0.576194411635519,
          "SampleQ1": 90.5,
          "SampleQ3": 91.3,
          "SampleSize": 5
        },
        {
          "Seqnr": 3,
          "SampleId": 240189655,
          "Value": 91.3,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T14:28:02Z",
          "SampleTimestampUtc": "2023-11-14T07:28:02Z",
          "CreatedTimestamp": "2023-11-14T14:28:02Z",
          "CreatedTimestampUtc": "2023-11-14T07:28:02Z",
          "UpdatedTimestamp": "2023-11-14T14:28:02Z",
          "UpdatedTimestampUtc": "2023-11-14T07:28:02Z",
          "NumViolations": 0,
          "SampleMean": 90.92,
          "SampleMin": 90.3,
          "SampleMax": 91.7,
          "SampleMedian": 90.8,
          "SampleStdev": 0.576194411635519,
          "SampleQ1": 90.5,
          "SampleQ3": 91.3,
          "SampleSize": 5
        },
        {
          "Seqnr": 2,
          "SampleId": 240189655,
          "Value": 90.3,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T14:28:02Z",
          "SampleTimestampUtc": "2023-11-14T07:28:02Z",
          "CreatedTimestamp": "2023-11-14T14:28:02Z",
          "CreatedTimestampUtc": "2023-11-14T07:28:02Z",
          "UpdatedTimestamp": "2023-11-14T14:28:02Z",
          "UpdatedTimestampUtc": "2023-11-14T07:28:02Z",
          "NumViolations": 0,
          "SampleMean": 90.92,
          "SampleMin": 90.3,
          "SampleMax": 91.7,
          "SampleMedian": 90.8,
          "SampleStdev": 0.576194411635519,
          "SampleQ1": 90.5,
          "SampleQ3": 91.3,
          "SampleSize": 5
        },
        {
          "Seqnr": 1,
          "SampleId": 240189655,
          "Value": 90.5,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T14:28:02Z",
          "SampleTimestampUtc": "2023-11-14T07:28:02Z",
          "CreatedTimestamp": "2023-11-14T14:28:02Z",
          "CreatedTimestampUtc": "2023-11-14T07:28:02Z",
          "UpdatedTimestamp": "2023-11-14T14:28:02Z",
          "UpdatedTimestampUtc": "2023-11-14T07:28:02Z",
          "NumViolations": 0,
          "SampleMean": 90.92,
          "SampleMin": 90.3,
          "SampleMax": 91.7,
          "SampleMedian": 90.8,
          "SampleStdev": 0.576194411635519,
          "SampleQ1": 90.5,
          "SampleQ3": 91.3,
          "SampleSize": 5
        },
        {
          "Seqnr": 5,
          "SampleId": 240189655,
          "Value": 90.8,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T14:28:02Z",
          "SampleTimestampUtc": "2023-11-14T07:28:02Z",
          "CreatedTimestamp": "2023-11-14T14:28:02Z",
          "CreatedTimestampUtc": "2023-11-14T07:28:02Z",
          "UpdatedTimestamp": "2023-11-14T14:28:02Z",
          "UpdatedTimestampUtc": "2023-11-14T07:28:02Z",
          "NumViolations": 0,
          "SampleMean": 90.92,
          "SampleMin": 90.3,
          "SampleMax": 91.7,
          "SampleMedian": 90.8,
          "SampleStdev": 0.576194411635519,
          "SampleQ1": 90.5,
          "SampleQ3": 91.3,
          "SampleSize": 5
        }
      ],
      "CFComment": "FOL",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "Y",
          "SpecLowEnabled": "Y",
          "spec_high": 105.0,
          "spec_low": 75.0,
          "target": 90.0
        },
        "ControlLimits": {
          "CtrlMeanLowEnabled": "N",
          "CtrlMeanTargetEnabled": "N",
          "CtrlMeanHighEnabled": "N",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "N",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "CtrlRangeLowEnabled": "N",
          "CtrlRangeHighEnabled": "N"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 5,
        "base_count": 5,
        "flagged_count": 0,
        "mean": 90.92,
        "median": 90.8,
        "stdev": 0.5153639490690063,
        "range": 1.4000000000000057,
        "min": 90.3,
        "max": 91.7,
        "q02": 90.316,
        "q05": 90.34,
        "q25": 90.5,
        "q75": 91.3,
        "q95": 91.62,
        "q98": 91.66799999999999,
        "NumViolations": 0,
        "Samples": "240189655"
      },
      "ParameterName": "CopperBallDiameterCTDownbond",
      "ParameterUnit": "um",
      "ChannelId": "70005",
      "SourceDataLevel": "C",
      "channel_name": "Ball_Diameter CU 30um CT_Downbond LSS Bump",
      "channel_description": "Bump Diameter, Downbond",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-11-14T07:28:02Z",
      "created_timestamp": "2023-11-14T14:28:02Z",
      "updated_timestamp_utc": "2023-11-14T07:28:02Z",
      "updated_timestamp": "2023-11-14T14:28:02Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-11-14T07:28:02Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:BAT:BE:BE1I:Wire_Bond1:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "BAT:BE:ZA332675B36:BE1I:Wire_Bond1:CopperBallDiameterCTDownbond:70005:240189655:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-11-14T14:28:02Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot::SPACEAGGED2:BAT:BE:BE1I:Wire_Bond1::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "BAT"
  }
}